app/mailers/vote_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class VoteMailer < ApplicationMailer
+  default from: 'from@example.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.